src/init/replace_vulnerability_detector.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright (C) 2023, Wazuh Inc.
+# Shell script that updates 'vulnerability-detector' to 'vulnerability-detection'
+# Author: Kevin Cardenas <kevincardenasmiranda9@gmail.com>
```

Comment:
Looking at the other scripts in this folder, it seems that only the scripts that are very old (+15 years old) include the author's name.

Reply:
should I change to something like?
```txt
# Copyright (C) 2015, Wazuh Inc.
# All rights reserved.
# Wazuh.com

# This program is free software; you can redistribute it
# and/or modify it under the terms of the GNU General Public
# License (version 2) as published by the FSF - Free Software
# Foundation.
```